src/datatablerow/datatablerow.jl

```diff
@@ -78,24 +73,6 @@ end
 isequal_colel(col::AbstractArray, r1::Int, r2::Int) =
     (r1 == r2) || isequal(Base.unsafe_getindex(col, r1), Base.unsafe_getindex(col, r2))
 
-isequal_colel(a::Any, b::Any) = isequal(a, b)
-isequal_colel(a::Nullable, b::Any) = !isnull(a) & isequal(unsafe_get(a), b)
-isequal_colel(a::Any, b::Nullable) = isequal_colel(b, a)
-isequal_colel(a::Nullable, b::Nullable) = isequal(a, b)
-
-# table columns are passed as a tuple of vectors to ensure type specialization
```

Comment:
Add this back, this was from https://github.com/JuliaData/DataTables.jl/pull/79/files#diff-bfe3bbcf63031d8faf171ed27ddac298R86